### ISSUE TITLE
Spirit Tools Problem

### ISSUE THREAD

**Issue by Kalmeira** [Xilef11/runesofwizardry-classics#17]
Spirirt Tools requires sacrifices different from what it states and will take more than it asks for. When provided with a golden pickaxe and 4 tnt, it says invalid sacrifice. when provided with glowstone, golden pickaxe, 4 tnt, and 18 levels, it works but only consumes 10 levels. Ifyou do not have 18 levels, it will consume the sacrifice and nothing happens. If you supply a golden sword, a golden pickaxe, 1 glowstone, and 4 tnt, it will consume everything and give you only a sword.

**Comment by Kalmeira** [comment]
Also, I am not sure if this is a bug, but the spirit pickaxe has a mining level of 0(wood) unless you are using the AOE mining feature. The spirit sword has infinite durability, the pickaxe doesn't.

**Comment by Xilef11** [comment]
Yes, these are bugs. I'll look into them.
I suppose you're using the latest 1.12.2 version?

**Comment by Kalmeira** [comment]
Yes, sorry. Runes of Wizardry 1.12.2-0.9.2 and Runes of Wizardry-Classics 1.12.2-0.5.8

**Comment by Xilef11** [comment]
Partially fixed in [aa51b8](https://github.com/Xilef11/runesofwizardry-classics/commit/aa51b83f4a220f3ab02d5b41fbfba3e2a9eb7f00)
Some of the sacrifice issues need to be fixed in the base mod. [54](https://github.com/Xilef11/Runes-of-Wizardry/issues/54)
